packages/api-client/src/main/client/RegisteredClient.ts

```diff
@@ -17,16 +17,20 @@
  *
  */
 
-interface RegisteredClient {
-  address?: string; // IP address
-  class: 'desktop' | 'phone' | 'tablet';
-  cookie: string; // Cookie label
-  id: string; // Client ID
+import {ClientClassification} from './ClientClassification';
+
+export interface RegisteredClient {
+  /** The IP address */
```

Comment:
Let's specify that it is the IP address from where the client has been registered.